Tests/webcomponents/mixins/ez-sandbox-yui-component.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>ez-subitem test</title>
+
+        <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="/Resources/public/webcomponents/mixins/ez-sandbox-yui-component.html">
+        <script>
+        window.addEventListener('WebComponentsReady', function () {
+            class TestElement extends eZ.SandboxYUIComponentMixin(Polymer.Element) {
+                static get is() {
+                    return 'ez-test-yui-component';
+                }
+            }
+            window.customElements.define(TestElement.is, TestElement);
+        });
+        </script>
+    </head>
+    <body>
+
+        <test-fixture id="BasicTestFixture">
+            <template>
+                <ez-test-yui-componentt></ez-test-yui-component>
```

Comment:
actually, it's because this is not needed... and then TestElement is not needed as well